webui/react/vite.config.mts

```diff
@@ -89,7 +90,7 @@ export default defineConfig(({ mode }) => ({
     'process.env.IS_DEV': JSON.stringify(mode === 'development'),
     'process.env.PUBLIC_URL': JSON.stringify((mode !== 'test' && publicUrl) || ''),
     'process.env.SERVER_ADDRESS': JSON.stringify(process.env.SERVER_ADDRESS),
-    'process.env.VERSION': '"0.37.1-dev0"',
+    'process.env.VERSION': JSON.stringify(process.env.VERSION || child.execSync('git describe --tags --always 2>/dev/null || echo "unknown"').toString().trimEnd()),
```

Comment:
should this look more like this from python?
```
            output = subprocess.run(["../version.sh"], capture_output=True)
```

Reply:
we compare this variable to the version number on master in order to show a prompt to the user indicating that there's an site update. Does this match the output of `version.sh`?

Reply:
I'll take a look. I think this particular line is slightly incorrect now (it would use a global list of git tags instead of a branch-sensitive one). Where does this comparison happen?

Reply:
https://github.com/determined-ai/determined/blob/main/webui/react/src/App.tsx#L99

Reply:
Okay, I did some digging, and this should be fine, although I did update the `process.env.VERSION` value to use `version.sh` instead of `git-describe`. As far as I can tell, the frontend uses the `VERSION` environment variable as a source of truth for its own version. There's a version stored somewhere in memory, in `DeterminedInfo.version` ([determinedInfo.tsx](https://github.com/determined-ai/determined/blob/a5d154133b67ba5b5c210bbbe542e92ef32deec5/webui/react/src/stores/determinedInfo.tsx#L66)), that is originally fetched from `process.env.VERSION`. After an update, though, `process.env.VERSION` will have moved, but the `DeterminedInfo.version` string will still be the previous version. The frontend will detect this as normal, because the frontend's canonical version always comes from `process.env.VERSION`.

I.e. the comparison should always work, because these version strings should all come from the same environment variable.